src/js/utils/underscore.js

```diff
@@ -736,6 +736,7 @@ _.result = function (object, property) {
 };
 
 const _isNumber = _.isNumber;
+const _isValidNumber = (val) => _isNumber(val) && !_isNaN(val);
 
-export { _isNaN, _isNumber };
+export { _isNaN, _isNumber, _isValidNumber };
 export default _;
```

Comment:
If you export the methods without the underscore prefix, we can eventually do that for all methods and remove the default export. Modules that want `_` will `import * as _ from 'underscore'`.